### PR TITLE
adding privacy and semantic wrappers

### PR DESCRIPTION
**Pull request by absaroj**

### CODE REVIEW

File: wrappers/swift/SDWSemanticContext.swift
Comment (eduardo-camacho) [inline wrappers/swift/SDWSemanticContext.swift:78]: typo "advertizing"

File: wrappers/swift/SDWSemanticContext.swift
Comment (eduardo-camacho) [inline wrappers/swift/SDWSemanticContext.swift:83]: typo

**Review by eduardo-camacho — APPROVED** [review]
Please make sure to get review from Lalit, Sid and Mannie

File: wrappers/swift/SemanticContext.swift
Comment (mannie) [inline wrappers/swift/SemanticContext.swift:116]: Do the parameters in the functions in this file need to be optionals?

File: wrappers/swift/PrivacyGuard.swift
Comment (lalitb) [inline wrappers/swift/PrivacyGuard.swift:11]: I am surely missing something here, but why do we need this constructor, and also the private instance variable in this class, when all the methods in the class are static. Also, the obj-c class being wrapped around also has only class methods.

File: wrappers/swift/PrivacyGuard.swift
Comment (mannie) [inline wrappers/swift/PrivacyGuard.swift:7]: Do we expect subclasses of this class? If not, we should mark this as `final` to prevent unexpected behaviors if someone chooses to subclass.

File: wrappers/swift/PrivacyGuardInitConfig.swift
Comment (mannie) [inline wrappers/swift/PrivacyGuardInitConfig.swift:6]: Do we expect subclasses of this class? If not, we should mark this as `final` to prevent unexpected behaviors if someone chooses to subclass.

File: wrappers/swift/PrivacyGuardInitConfig.swift
Comment (mannie) [inline wrappers/swift/PrivacyGuardInitConfig.swift:7]: This value is never changed; we should make this a constant instead.

File: wrappers/swift/PrivacyGuardInitConfig.swift
Comment (mannie) [inline wrappers/swift/PrivacyGuardInitConfig.swift:20]: If we make the underlying variable a public constant, we can remove this function. That's the Swift way of implementing accessors.

File: wrappers/swift/SemanticContext.swift
Comment (mannie) [inline wrappers/swift/SemanticContext.swift:7]: Do we expect subclasses of this class? If not, we should mark this as `final` to prevent unexpected behaviors if someone chooses to subclass.

File: wrappers/swift/SemanticContext.swift
Comment (mannie) [inline wrappers/swift/SemanticContext.swift:8]: `ODWSemanticContext` is a reference type right? If so, we should change this to `private var context: ODWSemanticContext`

File: wrappers/swift/SemanticContext.swift
Comment (mannie) [inline wrappers/swift/SemanticContext.swift:64]: nit: these `self.`s are not needed

File: wrappers/swift/PrivacyGuard.swift
Comment (absaroj) [reply wrappers/swift/PrivacyGuard.swift:11]: I think you are right here. Currently the constructor for this is not needed as all methods are static. removed it.

File: wrappers/swift/PrivacyGuardInitConfig.swift
Comment (absaroj) [reply wrappers/swift/PrivacyGuardInitConfig.swift:6]: It does have some instance method so it might be potentially subclassed in future. so keep it non-final.

File: wrappers/swift/SemanticContext.swift
Comment (absaroj) [reply wrappers/swift/SemanticContext.swift:7]: It has instance methods so potentially might be some advantage to subclassing in future. keeping it non-final.

File: wrappers/swift/SemanticContext.swift
Comment (absaroj) [reply wrappers/swift/SemanticContext.swift:8]: i kept the variable name to reflect that it is objc class instance called by the wrapper.

**Review by lalitb — APPROVED** [review]

**Review by mannie — APPROVED** [review]

**Comment by lalitb** [comment]
force merge to unblock. The CI is failing for older MacOS runners, not related to the change.

**Pull request merged**
